installer/src/main.rs

```diff
@@ -0,0 +1,44 @@
+use std::{env, fs, io::{stdin, stdout, Write}};
+
+use crate::{dotnet::check_dotnet, steam::check_steam};
+
+mod dotnet;
+mod steam;
+
+const NAME: &str = env!("CARGO_PKG_NAME");
+const VERSION: &str = env!("CARGO_PKG_VERSION");
+
+// TODO better handling of errors - print them into a log and exit instead of panic!
+// TODO app icon
+fn main() {
+    let dotnet_version = 7;
+
+    println!("{} {}", NAME, format!("v{}", VERSION));
+
+    // TODO GOG support
+    let mut terraria_path = check_steam();
+    println!("{}", terraria_path.display());
```

Comment:
GOG Galaxy and some other launchers should be checked, otherwise it's up to the user.